tests/StackExchange.Redis.Tests/TestBase.cs

```diff
@@ -361,14 +361,7 @@ protected IServer GetAnyMaster(IConnectionMultiplexer muxer)
         }
 
         public static string Me([CallerFilePath] string filePath = null, [CallerMemberName] string caller = null) =>
-#if NET472
-            "net472-"
-#elif NETCOREAPP3_1
-            "netcoreapp3.1-"
-#else
-            "unknown-"
-#endif
-         + Path.GetFileNameWithoutExtension(filePath) + "-" + caller;
+            Environment.Version.ToString() + Path.GetFileNameWithoutExtension(filePath) + "-" + caller;
```

Comment:
This isn't a 1:1 replacement, but the goal is to simple make it unique per version - let's go simpler and never update this with TFMs again.